packages/main/src/i18n/messagebundle.properties

```diff
@@ -208,6 +208,9 @@ MESSAGE_STRIP_INFORMATION=Information Bar
 #XFLD: MultiComboBox dialog button
 MULTICOMBOBOX_DIALOG_OK_BUTTON=OK
 
+#XMSG: Text used for reporting that a radio group requires one of the radio buttons to be checked
+RADIO_BUTTON_GROUP_REQUIRED=Please select one of these options.
```

Comment:
Better to avoid the overuse of "please". Also, if there are options that are not selectable (disabled state):
RADIO_BUTTON_GROUP_REQUIRED=Select one of the available options.